server/src/server.ts

```diff
@@ -23,9 +23,13 @@ import { WorkspaceEdit } from "vscode-languageserver";
 
 interface extensionConfiguration {
   askToStartBuild: boolean;
+  autoRunCodeAnalysis: boolean;
+  inlayHints: boolean;
 }
 let extensionConfiguration: extensionConfiguration = {
   askToStartBuild: true,
+  autoRunCodeAnalysis: false,
```

Comment:
What does this change imply?
This config is pre-existing and seemed to work. How is this related to the existing way of reading this configuration? Do we have 2 ways now? And when do they fire? And which one wins? And is one of them redundant?